### ISSUE TITLE
Attaching an already attached disk gives incorrect error message

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1204]
Originally reported of Google Code with ID 1151.

```
gnt-cluster --version
gnt-cluster (ganeti v2.16.0beta1-318-ga395b91) 2.18.0~alpha1

<b>What steps will reproduce the problem?</b>
1. Have an instance I1 with disk D1 and another instance I2
2. Try to attach disk D1 to instance I2 with:
   gnt-instance modify --disk 0:attach,uuid=*UUID_OF_DISK_D1* *INSTANCE_I2*

<b>What is the expected output? What do you see instead?</b>

Currently, I see the following error:
Unhandled Ganeti error: Disk *UUID_OF_DISK_D1* already attached to instance Just *INSTANCE_I2*

Instead it should be:

Unhandled Ganeti error: Disk *UUID_OF_DISK_D1* already attached to instance Just *INSTANCE_I1*
```
Originally added on 2015-11-30 13:27:57 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
The "Just" should of course not be present in the error message, so it should be:
Unhandled Ganeti error: Disk *UUID_OF_DISK_D1* already attached to instance *INSTANCE_I1*

This is, however, already fixed on stable-2.16 with:


commit bcdd69234880400942d6b5062c5469f3ea3e4b8d
Author: Lisa Velden <velden@google.com>
Date:   Mon Nov 23 15:42:09 2015 +0100

    Use only string value in error message
    
    Signed-off-by: Lisa Velden <velden@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>
```
Originally added on 2015-11-30 13:30:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:

commit ceb09b5326d773fcff2e8a114dbc28b7bd46b57f
Author: Lisa Velden <velden@google.com>
Date:   Mon Dec 14 15:13:10 2015 +0100

    Fix error message in attachInstanceDiskChecks
    
    Name the instance where disks are already attached to, which is not
    necessarily the instance where we want to attach a disk to.
    This fixes issue 1151.
    
    Signed-off-by: Lisa Velden <velden@google.com>
    Reviewed-by: Oleg Ponomarev <oponomarev@google.com>
```
Originally added on 2015-12-14 14:47:38 +0000 UTC.

Mentioned Issue with GoogleCode ID 1151 has been migrated to GitHub with ID #1204.
Changed State: Fixed
Added to Milestone: Release2.16
